### PR TITLE
feat(MessageLogger): Add delete & edit toggle

### PR DESCRIPTION
**Pull request by hypergonial**
Add the ability to toggle the logging of edits or deletes seperately. Quick testing seems to show that it works but I probably missed things.

P.S. I have no idea what I'm doing, feedback is welcome!

P.P.S. Should I add myself to the authors for small changes like these?

### CODE REVIEW

**Comment by hypergonial** [comment]
> perhaps it could be done as a runtime check in `shouldIgnore`? that way the user doesn't have to restart after toggling?

Good point, done!

File: src/plugins/messageLogger/index.tsx [hunk 1]
```diff
@@ -197,7 +207,7 @@ export default definePlugin({
         return cache;
     },
 
-    shouldIgnore(message: any) {
+    shouldIgnore(message: any, is_edit: boolean = false) {
```
Comment (Vendicated) [inline src/plugins/messageLogger/index.tsx:210]:
```suggestion
    shouldIgnore(message: any, isEdit: boolean = false) {
```

keep casing consistent! in javascript we use camelCase

File: src/plugins/messageLogger/index.tsx [hunk 2]
```diff
@@ -197,7 +207,7 @@ export default definePlugin({
...
         const { ignoreBots, ignoreSelf, ignoreUsers, ignoreChannels, ignoreGuilds } = Settings.plugins.MessageLogger;
```
Comment (Vendicated) [inline src/plugins/messageLogger/index.tsx:211]: add additional destructures here instead of calling them on the full path object below

Comment (hypergonial) [reply src/plugins/messageLogger/index.tsx:211]: I'm not entirely sure what you mean by this, sorry. Can you clarify?

Comment (Vendicated) [reply src/plugins/messageLogger/index.tsx:211]:
```suggestion
        const { ignoreBots, ignoreSelf, ignoreUsers, ignoreChannels, ignoreGuilds, logEdits, logDeletes } = Settings.plugins.MessageLogger;
```

Comment (hypergonial) [reply src/plugins/messageLogger/index.tsx]: Done, thanks!

**Comment by Vendicated** [comment]
thank you!

**Pull request merged**